build.gradle.kts

```diff
@@ -76,6 +73,20 @@ subprojects {
             dependsOn(tasks.named<DefaultTask>("assemble"))
         }
 
+        plugins.withId("java") {
+            //create a task to publish our sources
+            tasks.register<Jar>("sourcesJar") {
```

Comment:
Now adays just need to set some options

https://github.com/aws/aws-xray-sdk-java/blob/master/build.gradle.kts#L78

Reply:
I originally had that but for some reason at launch it didn't publish javadoc JARs. Will investigate and try to revert on my next PR.